datadog_checks_dev/datadog_checks/dev/tooling/e2e/platform.py

```diff
@@ -0,0 +1,6 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+MAC = 'mac'
+WINDOWS = 'windows'
+LINUX = 'linux'
```

Comment:
Let's remove this file, too much indirection

Reply:
I think its useful to keep for the sake of not writing `mac`/`windows` everywhere and avoid spelling mistakes/capitalization issues.